docs/usage/key-concepts/scheduling.md

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
@rarkins I'm ready for a first review of the `key-concept-scheduling` file and the related changes. Feel free to make your own changes/commits how you see fit to improve the text.

I'll stay away from this branch until you tell me I'm good to make further changes on it. That way we don't get in each others way. 😄